### PR TITLE
Run.sh: allow clean termination from a child run script

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: container/root/run.sh [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+  if [[ $STATUS != 0 ]]
+  then
+    echo "[run.d] failed executing - ${file}"
+    exit $STATUS
+  fi
+
 done
 
-echo "[nginx] starting (foreground)"
+echo "[nginx] start (foreground)"
```
Comment (bryanlatten) [inline container/root/run.sh]: just because someone asked me if it was stalled on `starting`

File: container/root/run.sh [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
-# Run shell scripts (ending in .sh) in run.d directory, if any
 for file in $RUN_SCRIPTS/*.sh; do
+
   echo "[run.d] executing ${file}"
+
   /bin/bash $file
+
+  STATUS=$?  # Captures exit code from script that was run
+
+  if [[ $STATUS == 1 ]]
```
Comment (ko-be) [inline container/root/run.sh]: I would pick a different exit code than 1, as exit code 1 has a special meaning.  See: http://tldp.org/LDP/abs/html/exitcodes.html

**Comment by Attamusc** [comment]
:+1:

**Comment by ko-be** [comment]
:+1:

**Pull request merged**